src/styles/partials/components/_table.scss

```diff
@@ -0,0 +1,88 @@
+/* BASIC TABLE STYLES */
+.table {
+  border: 1px solid $gray-light;
+  border-collapse: collapse;
+  color: #4c4c4c;
```

Comment:
this color needs to come from the variables file. If it isn't in the color list, we need to talk to sam.

Reply:
All fugitive hex colors have been replaced with their corresponding variables. Will include update in next commit.